CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Rama Changelog
+All notable changes to this project will be documented in this file.
+
+## [v0.1.0] 2021-06-18
```

Comment:
a timestamp is needless here, git will help us for the history

Reply:
removed